### PR TITLE
Add pygame GUI for Cliffwalking

### PR DESCRIPTION
**Pull request by younik**
Screenshot from 2022-07-25 15-39-37

### CODE REVIEW

**Review by pseudo-rnd-thoughts — CHANGES_REQUESTED** [review]
This generally looks good. I realised that this is not done already, but would it be helpful to have all of the images start with `cliffwalking-` to help future work, know which image is for which envionment

File: gym/envs/toy_text/cliffwalking.py [hunk 1]
```diff
@@ -24,15 +24,16 @@ class CliffWalkingEnv(Env):
     by Sutton and Barto](http://incompleteideas.net/book/bookdraft2018jan1.pdf).
 
     With inspiration from:
-    https://github.com/dennybritz/reinforcement-learning/blob/master/lib/envs/cliff_walking.py
+    [https://github.com/dennybritz/reinforcement-learning/blob/master/lib/envs/cliff_walking.py]
+    (https://github.com/dennybritz/reinforcement-learning/blob/master/lib/envs/cliff_walking.py)
```
[the remark this answers is not in the document]
Comment (younik) [reply gym/envs/toy_text/cliffwalking.py:28]: Yes, if you look here: https://www.gymlibrary.ml/environments/toy_text/cliff_walking/
you can see the link as text but it is not clickable; this change makes it clickable

**Comment by younik** [comment]
>This generally looks good. I realised that this is not done already, but would it be helpful to have all of the images start with cliffwalking- to help future work, know which image is for which environment

Yeah, it can be helpful, but some of the images are shared across environments (elf and stool)

**Comment by pseudo-rnd-thoughts** [comment]
LGTM

File: gym/envs/toy_text/cliffwalking.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pseudo-rnd-thoughts) [reply gym/envs/toy_text/cliffwalking.py:28]: Im stupid ignore

**Pull request merged**